lighthouse-core/audits/seo/tap-targets.js

```diff
@@ -248,13 +248,8 @@ function targetToTableNode(target) {
   const boundingRect = getBoundingRect(target.clientRects);
 
   return {
-    type: 'node',
-    lhId: target.node.lhId,
-    snippet: target.node.snippet,
-    path: target.node.devtoolsNodePath,
-    selector: target.node.selector,
+    ...Audit.makeNodeItem(target.node),
     boundingRect,
```

Comment:
Which boundingRect should be used here? Using the boundingRect of all clientRects goes back to the first commit of #10716, but `getNodeDetails` also didn't exist back then, so that might have just been the most convenient way to do it at the time. Should it be the just `node.boundingRect`? When can the two differ? `getBoundingClientRect` should normally contain all child rects, but does it ever differ (transforms or something?)

Reply:
I'll followup with this in a different PR.